sdk/src/accounts/customUserAccountSubscriber.ts

```diff
@@ -0,0 +1,53 @@
+import { DataAndSlot, UserAccountEvents, UserAccountSubscriber } from './types';
+import { PublicKey } from '@solana/web3.js';
+import StrictEventEmitter from 'strict-event-emitter-types';
+import { EventEmitter } from 'events';
+import { UserAccount } from '../types';
+
+export class CustomUserAccountSubscriber implements UserAccountSubscriber {
```

Comment:
should we call it Fixed, Static, Constant or Mock UserAccountSubscriber 🤔

Reply:
mock's good 👍